.github/workflows/on-merge-to-main.yml

```diff
@@ -38,3 +38,103 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           TAG_PREFIX: v
           DEFAULT_BUMP: ${{ github.event.inputs.bump_type || 'patch' }}
+
+  build:
+    needs: versioning
+    if: needs.versioning.outputs.new_tag
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v4
+        with:
+          lfs: true
+          ref: ${{ needs.versioning.outputs.new_tag }}
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+      - name: Setup Node.js
+        uses: actions/setup-node@v5
+        with:
+          node-version: 23
```

Comment:
We're on 22 (LTS) right now. Better to use [node-version-file](https://github.com/actions/setup-node?tab=readme-ov-file#usage), right?

Reply:
Done